New package: Remeha.PCServiceTool version 7.3.8.0

TutkuSilaVuralEyovgeAlten opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135318)

TutkuSilaVuralEyovgeAlten committed:
New package: Remeha.PCServiceTool version 7.3.8.0

TutkuSilaVuralEyovgeAlten commented:
@microsoft-github-policy-service agree [company="Remeha BV"]

TutkuSilaVuralEyovgeAlten commented:
> @TutkuSilaVuralEyovgeAlten please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

 @microsoft-github-policy-service agree company="Remeha"

TutkuSilaVuralEyovgeAlten commented:
@microsoft-github-policy-service agree company="Remeha"

Trenly commented:
@wingetbot run

Trenly commented on manifests/r/Remeha/PCServiceTool/7.3.8.0/Remeha.PCServiceTool.installer.yaml:
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Remeha.PCServiceTool
```
```suggestion
PackageIdentifier: Remeha.ServiceTool
```
Package already exists under different identifier; Existing directory should be used

Trenly commented on manifests/r/Remeha/PCServiceTool/7.3.8.0/Remeha.PCServiceTool.locale.en-US.yaml:
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Remeha.PCServiceTool
```
```suggestion
PackageIdentifier: Remeha.ServiceTool
```
Package already exists under different identifier; Existing directory should be used

Trenly commented on manifests/r/Remeha/PCServiceTool/7.3.8.0/Remeha.PCServiceTool.yaml:
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Remeha.PCServiceTool
```
```suggestion
PackageIdentifier: Remeha.ServiceTool
```
Package already exists under different identifier; Existing directory should be used

TutkuSilaVuralEyovgeAlten committed:
Update manifests/r/Remeha/PCServiceTool/7.3.8.0/Remeha.PCServiceTool.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

TutkuSilaVuralEyovgeAlten committed:
Update manifests/r/Remeha/PCServiceTool/7.3.8.0/Remeha.PCServiceTool.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

TutkuSilaVuralEyovgeAlten committed:
Update manifests/r/Remeha/PCServiceTool/7.3.8.0/Remeha.PCServiceTool.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

TutkuSilaVuralEyovgeAlten committed:
Update Remeha.PCServiceTool.installer.yaml

TutkuSilaVuralEyovgeAlten committed:
Update Remeha.PCServiceTool.locale.en-US.yaml

TutkuSilaVuralEyovgeAlten committed:
Update Remeha.PCServiceTool.yaml

TutkuSilaVuralEyovgeAlten committed:
Update

Updated folder name according to https://github.com/microsoft/winget-pkgs/pull/135318#issuecomment-1904104752

stephengillie commented:
I'm not seeing the cause for the [Policy 1.2](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#12-security) check.

stephengillie commented:
@wingetbot waivers Add Policy-Test-1.2
